### DISCUSSION TITLE
New Style :O

### DISCUSSION THREAD

**Discussion by ghost** [akai-katto/dandere2x#177] [Show and tell]
Yeah. I just redid a bit of the gui so that it looks a bit nicer.

![1](https://user-images.githubusercontent.com/58604290/104851320-992c1680-58f4-11eb-8714-32ab9d073f68.png)

If you want to have a look at it or even want to change it, here is the fork: https://github.com/Harutami/dandere2x

**Comment by akai-katto** [comment]
It seems microsoft considers my own dandere2x binary and your binary a trojan.

**Reply by ghost** [reply]
Yeah its because its not signed as microsoft wants it to. I could look into it and see if its possible if that can be changed

My antivirus also deleted Dandere when I forgot to give it administrator privilages, because it tries to write on a directory without admin. So I always needed to get a new copy of Dandere lmao.

**Reply by akai-katto** [reply]
): 

I inferred there's a lot of "trojan-like' behavior that dandere2x probably does, i.e grabbing files from the internet, unzipping files, deleting workspaces, etc. I wasn't really sure if there was a way around this. I should reach out to a larger group to ask about doing this.

**Comment by ghost** [comment]
hmmm. well this popup occurs before it even started so my first guess was the building of the binaries. It could also be some module or line of code which would be hassle to find

**Reply by ghost** [reply]
> hmmm. well this popup occurs before it even started so my first guess was the building of the binaries. It could also be some module or line of code which would be hassle to find

Yeah as I searched, it is the problem of signing the application. But to get such a certificate you need to buy one as a company which is not in any way possible with open source projects.

**Reply by ghost** [reply]
Except someone creates a company and has money to buy such a certificate.

I don't know how it is on other peoples pc's but if I start Dandere as administrator it doesn't show up so yeah I think we need to live with that
